tests/unit_tests/olio/test_wellspec_keywords.py

```diff
@@ -47,172 +46,98 @@ def test_check_value_known_keyword(keyword, value, boolean_return_expected):
-    })
+    well_data = test_well_dataframe[column_list]
 
     # Act
-    well_dict = wk.load_wellspecs(wellspec_file, well, column_list)
+    well_dict = wk.load_wellspecs(wellspec_file_one_well, well, column_list)
 
     # Assert
     assert len(well_dict) == 1
     pd.testing.assert_frame_equal(well_dict["TEST_WELL"], well_data)
 
 
-def test_load_wellspecs_column_list_none(tmp_path):
+def test_load_wellspecs_column_list_none(wellspec_file_two_wells):
     # Arrange
-    wellspec_file = f"{tmp_path}/test.dat"
     well = None
     column_list = None
 
-    with open(wellspec_file, "w") as file:
-        file.write("""
-WELLSPEC TEST_WELL1
-IW    JW    L    KH    RADW    SKIN    RADB    WI    STAT    LENGTH    ANGLV    ANGLA    DEPTH
-18    28    2    NA    0.320   0.000   NA      NA    ON      5.000     88.080   86.800   9165.280
-18    28    3    NA    0.320   0.000   NA      NA    ON      5.000     88.080   86.800   9165.280
-18    28    4    NA    0.320   0.000   NA      NA    ON      5.000     88.080   86.800   9165.280
-18    28    5    NA    0.320   0.000   NA      NA    ON      5.000     88.080   86.800   9165.280
-
-WELLSPEC TEST_WELL2
-IW    JW    L    KH    RADW    SKIN    RADB    WI    STAT    LENGTH    ANGLV    ANGLA    DEPTH
-23    52    9    NA    0.260   0.000   NA      NA    ON      4.000     89.240   88.230   5492.600
-23    52    10   NA    0.260   0.000   NA      NA    ON      4.000     89.240   88.230   5492.600
-23    52    11   NA    0.260   0.000   NA      NA    ON      4.000     89.240   88.230   5492.600
-23    52    12   NA    0.260   0.000   NA      NA    ON      4.000     89.240   88.230   5492.600
-            """)
-
     # Act
-    well_dict = wk.load_wellspecs(wellspec_file, well, column_list)
+    well_dict = wk.load_wellspecs(wellspec_file_two_wells, well, column_list)
 
     # Assert
     assert len(well_dict) == 2
     assert all([value is None for value in well_dict.values()])
 
 
-def test_load_wellspecs_all_null(tmp_path):
+def test_load_wellspecs_all_null(wellspec_file_null_well):
     # Arrange
-    wellspec_file = f"{tmp_path}/test.dat"
     well = None
     column_list = []
 
-    with open(wellspec_file, "w") as file:
-        file.write("""
-WELLSPEC TEST_WELL
-IW    JW    L    KH    RADW    SKIN    RADB    WI    STAT    LENGTH    ANGLV    ANGLA    DEPTH
-NA    NA    NA   NA    NA      NA      NA      NA    NA      NA        NA       NA       NA
-NA    NA    NA   NA    NA      NA      NA      NA    NA      NA        NA       NA       NA
-            """)
-
     # Act
-    well_dict = wk.load_wellspecs(wellspec_file, well, column_list)
+    well_dict = wk.load_wellspecs(wellspec_file_null_well, well, column_list)
 
     # Assert
     assert well_dict == {}
+
+
+def test_get_well_pointers(wellspec_file_two_wells):
+    # Act
+    well_pointers = wk.get_well_pointers(wellspec_file_two_wells)
+
+    # Assert
+    assert len(well_pointers) == 2
+    assert well_pointers["TEST_WELL1"] == 21
```

Comment:
These pointer values could potentially be system dependent.  I'm okay for the test to go in like this – if it runs on my Mac and on the hpc then we are probably okay, though it would be good to execute the tests on a Windows laptop and on Azure at some point.  Don't worry about it just now though.